pynucastro/library/tabular/README.md

```diff
@@ -10,6 +10,7 @@ rates are tabulated in a two-dimensional space of:
 This directory includes tabulated rates obtained from Suzuki et al.,
 2016, ApJ 817:163, downloaded from
 https://www.phys.chs.nihon-u.ac.jp/laboratory/suzuki
```

Comment:
This link still works, and is in English (I think it's still good to keep the wayback machine link around, though)
```suggestion
https://www.phys.chs.nihon-u.ac.jp/suzuki/data2/
```